<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LampsClient</class>
 <widget class="QMainWindow" name="LampsClient">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>414</width>
    <height>559</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LampsClient</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="labelIP">
      <property name="minimumSize">
       <size>
        <width>64</width>
        <height>32</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>64</width>
        <height>32</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>ip</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <spacer name="hs01">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Minimum</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>64</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="2">
     <widget class="QLineEdit" name="lineEditIP">
      <property name="minimumSize">
       <size>
        <width>256</width>
        <height>32</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>256</width>
        <height>32</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="inputMask">
       <string/>
      </property>
      <property name="maxLength">
       <number>15</number>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Minimum</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>23</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="labelPort">
      <property name="minimumSize">
       <size>
        <width>64</width>
        <height>32</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>64</width>
        <height>32</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>port</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <spacer name="hs02">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Minimum</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>64</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="2">
     <widget class="QSpinBox" name="spinBoxPort">
      <property name="minimumSize">
       <size>
        <width>256</width>
        <height>32</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>256</width>
        <height>32</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="minimum">
       <number>10000</number>
      </property>
      <property name="maximum">
       <number>65535</number>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Minimum</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>22</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="labelStatus">
      <property name="minimumSize">
       <size>
        <width>64</width>
        <height>32</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>64</width>
        <height>32</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 0, 0);</string>
      </property>
      <property name="text">
       <string>X</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <spacer name="hs03">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Minimum</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>64</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="4" column="2">
     <widget class="QPushButton" name="pushButtonConnect">
      <property name="minimumSize">
       <size>
        <width>256</width>
        <height>32</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>256</width>
        <height>32</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Connect</string>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <spacer name="verticalSpacer_3">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Minimum</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>23</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="6" column="2">
     <widget class="QComboBox" name="lampsList">
      <property name="minimumSize">
       <size>
        <width>256</width>
        <height>32</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>256</width>
        <height>32</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <bold>true</bold>
       </font>
      </property>
     </widget>
    </item>
    <item row="7" column="2">
     <spacer name="verticalSpacer_4">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Minimum</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>22</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="8" column="2">
     <widget class="QPushButton" name="pushButtonOn">
      <property name="minimumSize">
       <size>
        <width>256</width>
        <height>32</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>256</width>
        <height>32</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>On</string>
      </property>
     </widget>
    </item>
    <item row="9" column="2">
     <spacer name="verticalSpacer_5">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Minimum</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>23</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="10" column="2">
     <widget class="QPushButton" name="pushButtonOff">
      <property name="minimumSize">
       <size>
        <width>256</width>
        <height>32</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>256</width>
        <height>32</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Off</string>
      </property>
     </widget>
    </item>
    <item row="11" column="2">
     <spacer name="verticalSpacer_6">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Minimum</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>22</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="12" column="2">
     <widget class="QPushButton" name="pushButtonAdd">
      <property name="minimumSize">
       <size>
        <width>256</width>
        <height>32</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>256</width>
        <height>32</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
    </item>
    <item row="13" column="2">
     <spacer name="verticalSpacer_7">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Minimum</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>23</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="14" column="2">
     <widget class="QPushButton" name="pushButtonRemove">
      <property name="minimumSize">
       <size>
        <width>256</width>
        <height>32</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>256</width>
        <height>32</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Remove</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>414</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>lineEditIP</tabstop>
  <tabstop>spinBoxPort</tabstop>
  <tabstop>pushButtonConnect</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
